pgmpy/independencies/Independencies.py

```diff
@@ -55,6 +55,9 @@ def __eq__(self, other):
         other_assertions = other.get_assertions()
         return all(self_independency in other_assertions for self_independency in self.get_assertions())
 
+    def __hash__(self):
+        return sum(hash(assertion) for assertion in self.get_assertions())
```

Comment:
@kshitij10496 I am not very sure this will be unique in each case. Why don't you simply convert `self.assertions` to a `str` and return the hash of that string.